include/loops/loops.hpp

```diff
@@ -33,6 +33,7 @@ enum {
     OP_STORE,
 
     OP_MOV,
+    OP_MOVK,   //Move bytes to shifted byte position of register and keep other bits unchanged.
```

Comment:
it's ARM-specific operation, right? Can you add a proper prefix to it? OP_AARCH64_MOVK or something like that

Reply:
Done.